web_monitoring/tests/test_diffing_server_exc_handling.py

```diff
@@ -0,0 +1,55 @@
+from tornado.testing import AsyncHTTPTestCase
+import web_monitoring.diffing_server as df
+from web_monitoring.diff_errors import UndecodableContentError
+
+class DiffingServerExceptionHandlingTest(AsyncHTTPTestCase):
+
+    def get_app(self):
+        app = df.make_app()
+        app.listen(str(self.get_http_port()))
+        return
+
+    def testInvalidURLaFormat(self):
+        response = self.fetch('http://localhost:'+str(self.get_http_port())+'/html_token?format=json&include=all&a=example.org&b=https://example.org', method='GET')
```

Comment:
Can you please use a format string for this? e.g:

```py
port = self.get_http_port()
response = self.fetch(
    f'http://localhost:{port}/html_token?format=json&include=all&a=example.org&b=https://example.org',
    method='GET')
```

Same for all the other tests here.